src/modules/search/saga.js

```diff
@@ -17,19 +17,28 @@ import {
   callApi,
   normalizeEntityResults,
 } from '../api/helpers';
+import { getFetchUnitsRequest } from '../unit/helpers';
 
 function* searchUnits({
   payload: { params },
 }: FetchAction): Generator<*, *, *> {
   let data = [];
+  let request = null;
+
   // Make search request only when there's input
   if (params.input && params.input.length) {
-    const request = createRequest(createUrl('search/', params));
-    const { bodyAsJson } = yield call(callApi, request);
-    data = bodyAsJson.results
-      ? normalizeEntityResults(bodyAsJson.results, new schema.Array(unitSchema))
-      : [];
+    request = createRequest(createUrl('search/', params));
+    // Otherwise get all units in order to comply with accessibility
+    // requirements
+  } else {
+    request = getFetchUnitsRequest(params);
```

Comment:
The search request does not return results on an empty result and I wasn't able to determine a query combination that would do so.

So instead of using the search endpoint, we find units by just directly requesting them from the unit endpoint. This is a clear hack, but I don't think that we can expect changes to the API anytime soon.